sdk/src/test/java/io/dapr/actors/runtime/ActorMethodInfoMapTest.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors.runtime;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.lang.reflect.Method;
+import java.lang.reflect.Parameter;
+import java.util.ArrayList;
+
+/**
+ * Unit tests for ActorMethodInfoMap.
+ */
+public class ActorMethodInfoMapTest {
+
+    @Test
+    public void normalUsage() {
+        ArrayList<Class<TestActor>> interfaceTypes = new ArrayList<Class<TestActor>>();
+        interfaceTypes.add(TestActor.class);
+        ActorMethodInfoMap m = new ActorMethodInfoMap(interfaceTypes);
+
+        try {
+            Method m1 = m.LookupActorMethodInfo("getData");
+            Assert.assertEquals("getData", m1.getName());
+            Class c = m1.getReturnType();
+            Assert.assertEquals(c.getClass(), String.class.getClass());
+            Parameter[] p = m1.getParameters();
+            Assert.assertEquals(p[0].getType().getClass(), String.class.getClass());
+        }
+        catch(Exception e) {
+            Assert.fail("Exception not expected.");
```

Comment:
No need to try catch it. Let the exception propagate, the test will fail as well with less code.

Reply:
Unfortunately that doesn't compile.  The only alternative I can think of is to mark `normalUsage()` itself with a `throws`, but that looks weird too.  I'm going to leave it, but if you know of a better way we can change it.